### PR TITLE
Greetings from api 

### PR DESCRIPTION
**Pull request by nwakauc**
## The following were implemented in this PR;

 - Created a new Rails app called 'hello-rails-react'.
 - Installed needed node dependencies.
 - Created a babel.config.js in the root folder.
 - Updated webpack.config.js to include the babel loader.
- Set up Postgres database
 - Created a table for storing messages and create 5 different greetings.
 - Created an API endpoint that selects a random greeting from the table.
 - Created a static view that is the root of the app.
 - Created the App component with react-router and rendered it in static view.
 - Created the Greeting component that will display a greeting. Set it up as a route in your App component.
 - Created a store, an action, and a reducer that will connect to the API endpoint to get the random greeting.
 - Display the random greeting in Greeting component.

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @uchexm,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,235 @@
...
+
+  - [Create a new API controller with its route config in routes.rb]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (t-yanick) [inline README.md:144]: - [ ] Great job working on this milestone. When I click on the MIT license link, I am re-directed to an error 404 page. Kidly fix this as it not professional to allow broken links in our README

File: README.md [hunk 2]
```diff
@@ -1,24 +1,235 @@
...
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (t-yanick) [inline README.md]: - [ OPTIONAL] 
This text is provided at part of the README template to guide on adding the MIT license. You can consider deleting it to make your README look clean

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @uchexm , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  App working as expected
:heavy_check_mark:  Required gems and packages used
:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @uchexm , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  App working as expected
:heavy_check_mark:  Required gems and packages used
:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Abenezer-Tilahun) [reply README.md:144]: Fixed.

File: README.md (hunk 2 above)
Comment (Abenezer-Tilahun) [reply README.md]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @uchexm 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git -Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**